packages/api/bin/backup.js

```diff
@@ -28,7 +28,7 @@ async function backup(table, region = 'us-east-1', folder = 'backups') {
     dynoParams.accessKeyId = 'my-access-key-id';
     dynoParams.secretAccessKey = 'my-secret-access-key';
     dynoParams.region = 'us-east-1';
-    dynoParams.endpoint = getLocalstackEndpoint('dynamodb');
+    dynoParams.endpoint = getLocalstackEndpoint('DynamoDB');
```

Comment:
Interesting. I missed the OG PR on this, I'm wondering why we chose to upcase this service in this way.